src/styles/main.scss

```diff
@@ -0,0 +1,46 @@
+.conteiner {
+  display: flex;
+  justify-content: center;
+  margin: 100px;
+}
+
+.arrow {
+  width: 10px;
+  border-radius: 5px;
+  transform-origin: bottom;
+}
+
+.arrow--minutes {
+  position: absolute;
+  background-color: red;
+  height: 150px;
+  margin-top: 50px;
+  animation: time 3600s linear  infinite;
+}
+
+@keyframes time {
+  from {
+    transform: rotate(0);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.arrow--second {
+  position: relative;
+  background-color: royalblue;
+  height: 200px;
+  animation: time 60s linear  infinite;
+}
+
+.circle {
+  z-index: 1;
+  position: absolute;
+  background-color: royalblue;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  margin: 190px 0 0 3px;
```

Comment:
why exactly this margins? if you use absolute position, use `top/left/right/bottom` styles